kong/templates/nginx_kong.lua

```diff
@@ -25,6 +25,7 @@ lua_shared_dict kong_core_db_cache          ${{MEM_CACHE_SIZE}};
 lua_shared_dict kong_core_db_cache_miss     12m;
 lua_shared_dict kong_db_cache               ${{MEM_CACHE_SIZE}};
 lua_shared_dict kong_db_cache_miss          12m;
+lua_shared_dict kong_log_level              1m;
```

Comment:
not trying to be stingy person as it's just 1m 😄 but why do we just use `kong`? since we are storing just one number or string.

Reply:
I've tried to use it but the tests failed 😞

Reply:
> not trying to be stingy person as it's just 1m 😄 but why do we just use `kong`? since we are storing just one number or string.

Agreed here.

> I've tried to use it but the tests failed 😞
@gruceo do we know why the tests failed?

Reply:
Found out why the tests failed. This shm is reset every time in the `init_by_lua` phase: https://github.com/Kong/kong/blob/master/kong/init.lua#L547

That's why this test fails: https://github.com/Kong/kong/blob/master/spec/02-integration/04-admin_api/22-debug_spec.lua#L459-L497

It does a `kong reload`, so the `init_by_lua` phase gets executed. To fix it, we'll need to find another way to write that test.